From a03458cf6c761fbfe0fd7745c54923d66f1918b4 Mon Sep 17 00:00:00 2001
From: Jerem1ah <1526100148@qq.com>
Date: Sat, 23 May 2026 00:22:42 +0800
Subject: [PATCH] exec anti-anti-frida.py

---
 src/embed-agent.py | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/embed-agent.py b/src/embed-agent.py
index cfb4a594..29d5bb5b 100644
--- a/src/embed-agent.py
+++ b/src/embed-agent.py
@@ -78,6 +78,14 @@ def main(argv):
                 shutil.copy(agent, embedded_agent)
             else:
                 embedded_agent.write_bytes(b"")
+            import os
+            custom_script=str(output_dir)+"/../../../../frida/subprojects/frida-core/src/anti-anti-frida.py"
+            return_code = os.system("python3 "+custom_script+" "+str(priv_dir / f"frida-agent-{flavor}.so"))
+            if return_code == 0:
+                print("anti-anti-frida finished")
+            else:
+                print("anti-anti-frida error. Code:", return_code)
+            
             embedded_assets += [embedded_agent]
     elif host_os in {"freebsd", "qnx"}:
         embedded_agent = priv_dir / "frida-agent.so"
-- 
2.53.0.windows.1

